imap_processing/idex/tests/test_l1_cdfs.py

```diff
@@ -0,0 +1,79 @@
+import copy
+import os
+
+import numpy as np
+import pytest
+import xarray as xr
+from cdflib.xarray import cdf_to_xarray
+
+from imap_processing.idex.idex_packet_parser import PacketParser
+
+
+@pytest.fixture(scope="session")
+def decom_test_data():
+    return PacketParser("imap_processing/idex/tests/imap_idex_l0_20230725_v01-00.pkts")
+
+
+def test_idex_cdf_file(decom_test_data):
+    # Verify that a CDF file can be created with no errors thrown by xarray_to_cdf
+    file_name = decom_test_data.write_l1_cdf(version="01", description="")
+    assert file_name == "imap_idx_l1_20250724_0000_p_v01.cdf"
+    os.remove(file_name)
+
+
+def test_bad_cdf_attributes(decom_test_data):
+    # Deliberately mess up the attributes to verify that no CDF is created
+    copy_to_modify = copy.deepcopy(decom_test_data)
```

Comment:
Rather than needing to deepcopy this, we should just update the fixture above to not be `session`, so it will create a new data file every single time if you need a fresh one.

Reply:
changed